slides/week0/week0.org

```diff
@@ -0,0 +1,484 @@
+#+TITLE: Intro to Programming
+#+AUTHOR: Jason Gibson
+#+EMAIL: jgibson37@gateche.edu
+
+* What are we doing today?
+- Instructor introductions
+- What is programming
+- Hello World!
+- Variables
+- Control Flow
+- Conditionals
+
+* Meet Jason
+#+ATTR_HTML: :width 15%
+[[file:https://i.imgur.com/izC5WWA.jpg]]
+- Jason Gibson
+  - Junior, Computer Science (Threads: Devices, Intelligence)
+  - Inside RoboJackets: Training Chair, IGVC Software Lead
+  - Outside RoboJackets: Avid lover of dad jokes
+  - Master of cringe worthy pictures
+- How to contact me
+  - Slack: [[https://robojackets.slack.com/messages/@jasongibson/][@jasongibson]]
+  - Email: [[mailto:jgibson37@gatech.edu][jgibson37@gatech.edu]]
+
+* What Programming Is
+#+BEGIN_NOTES
+Talk for a minute or two about everything programming can do
+#+END_NOTES
+
+* What Programming Is Not
+#+BEGIN_NOTES
+This is meant to emphasize that they will not become a God programmer overnight
+and to dispell the misconception that they are going to be coding amazing things
+immediately.
+#+END_NOTES
+- Impossible
+- Easy
+
+* Programming Failures
```

Comment:
The Mars Climate Orbiter and Ariane 5 incidents were both programming errors, but (imo) mistyping a terminal command is not.

I'm not familiar with the RoboCup incident.